src/index.js

```diff
@@ -4,21 +4,66 @@ import './style.css';
 const form = document.querySelector('.form-input');
 const [name, score] = form.elements;
 const objPlayers = new Players();
+const btnRefresh = document.querySelector('.refresh');
 
 if (localStorage.savedPlayers) {
   objPlayers.players = JSON.parse(localStorage.getItem('savedPlayers'));
 }
 
-form.addEventListener('submit', (e) => {
+/* Creating a new game
+const keyNewGameAPIs = async () => {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+    method: 'POST',
+    body: JSON.stringify({ name: "Nestor's cool name" }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
+*/
```

Comment:
- **[OPTIONAL]** Remove the unnecessary commented code remember YAGNI principle for better practices of JavaScript. 💯